src/Builder/Expression/FactoryTrait.php

```diff
@@ -970,14 +970,13 @@ public static function isArray(
      * New in MongoDB 4.4.
      *
      * @see https://www.mongodb.com/docs/manual/reference/operator/aggregation/isNumber/
-     * @no-named-arguments
-     * @param ExpressionInterface|Type|array|bool|float|int|non-empty-string|null|stdClass ...$expression
+     * @param ExpressionInterface|Type|array|bool|float|int|non-empty-string|null|stdClass $expression
      */
     public static function isNumber(
-        Type|ExpressionInterface|stdClass|array|bool|float|int|null|string ...$expression,
+        Type|ExpressionInterface|stdClass|array|bool|float|int|null|string $expression,
```

Comment:
What triggered both changes to the generated `FactoryTrait.php` and `IsNumberOperator.php` files?

The only modification to `isNumber.yaml` was new tests.

Reply:
Ah, I missed that when reviewing the diff. Thanks for clarifying!